internal/servers/root_certificate.go

```diff
@@ -11,13 +11,46 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+// The CertificateAuthority id will always be set to "roots".
+// The const ca_id contains this value
+const ca_id = "roots"
```

Comment:
mind adding godoc comments for exported/public things like `CA_ID` and `RootCertificateKeys`?